alns/stopping_criteria/MaxRuntime.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from alns.State import State
+from alns.stopping_criteria.StoppingCriterion import StoppingCriterion
+
+
+class MaxRuntime(StoppingCriterion):
+    def __init__(self, max_runtime: int) -> None:
+        """
+        Criterion that stops after a maximum number of iterations.
+        """
+        if max_runtime < 0:
+            raise ValueError("Max iterations must be non-negative.")
+
+        self._max_runtime = max_runtime
+        self._elapsed_runtime = None
+        self._start_runtime = None
+
+    @property
+    def max_runtime(self) -> float:
+        return self._max_runtime
+
+    @property
+    def elapsed_runtime(self) -> float:
+        return self._elapsed_runtime
```

Comment:
Issue: mypy complains about this because the values are default None, which is not a float.

You can fix this by typing them as `Optional[float]` (`from typing import Optional`).